addon/styles/addon.scss

```diff
@@ -29,14 +36,32 @@
     flex-direction: row;
     justify-content: flex-end;
     align-items: center;
-
+    width: calc(100% - 225px);
+    background: rgba(255,255,255,0.95);
+    bottom: 0px;
+    box-shadow: 0 8px 8px -8px #696868, 0 -8px 8px -8px #696868;
+    position: absolute;
     .frost-button,
     .frost-link {
       margin-left: 5px;
     }
+    &::before{
+      content: counter(selected) " Items selected";
```

Comment:
Ok cool - for accessibility across devs I think it's good practice to adhere to standard styles of development when possible - definitely not a big deal, just a general note